services/drone/drone-build.tester.js

```diff
@@ -0,0 +1,40 @@
+'use strict'
+
+const Joi = require('joi')
+const { isBuildStatus } = require('../build-status')
+const { ServiceTester } = require('../tester')
+
+const t = (module.exports = new ServiceTester({
+  id: 'drone-build',
+  title: 'Drone io',
+  pathPrefix: '/drone',
+}))
+
+// Drone Build
+
+t.create('build status on default branch')
+  .get('/build/drone/drone.json')
+  .expectBadge({
+    label: 'build',
+    message: Joi.alternatives().try(isBuildStatus, Joi.equal('none')),
+  })
+
+t.create('build status on named branch')
+  .get('/build/drone/drone/master.json')
+  .expectBadge({
+    label: 'build',
+    message: Joi.alternatives().try(isBuildStatus, Joi.equal('none')),
+  })
+
+t.create('unknown repo')
+  .get('/build/this-repo/does-not-exist.json')
+  .expectBadge({ label: 'build', message: 'none' })
+
+t.create('invalid svg response')
+  .get('/build/foo/bar.json')
+  .intercept(nock =>
+    nock('https://cloud.drone.io/api')
+      .get('/foo/bar.svg')
```

Comment:
FWIW though, I think the reason this is failing is because it's missing the `badges` part of the url path here, so nock isn't matching the actual request (`https://cloud.drone.io/api/badges/foo/bar.svg`

Reply:
Perfect, thanks for the quick reply/help! Overlooked that one. I'll keep it in and fix it now!

Reply:
I'd still suggest removing it. I know the Travis badge tester had this, but it's not a crucial test and not one typically added (see the testers for Codeship, Codacy, Beerpay, Gitlab Pipeline, Readthedocs, Snyk, etc.)

Reply:
Hmm, the actual URL being sent out is:

```
https://cloud.drone.io/api/badges/foo/bar/status.svg
```

I changed both the `api/badges` and `foo/bar/status.svg` part. But still, the test is failing. Do you have any additional tips/tricks?

Reply:
Sorry, didn't see your previous comment. I'll remove it!